client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -53,7 +52,6 @@ class CancelPurchase extends React.Component {
 		includedDomainPurchase: PropTypes.object,
 		purchase: PropTypes.object,
 		purchaseId: PropTypes.number.isRequired,
-		selectedSite: PropTypes.oneOfType( [ PropTypes.bool, PropTypes.object ] ),
```

Comment:
This looks like it's still used one place in the class, so should it really be removed from the prop list?

Reply:
Same thing here.. removed the whole thing thinking that's what made it optional, didn't realize it was already optional because it lacked `.isRequired`.